examples/chef/common/clusters/media-input/MediaInputManager.cpp

```diff
@@ -51,16 +53,32 @@ CHIP_ERROR MediaInputManager::HandleGetInputList(chip::app::AttributeValueEncode
 
 uint8_t MediaInputManager::HandleGetCurrentInput()
 {
-    return mCurrentInput;
+    uint8_t currentInput = 1;
+    Status status        = Attributes::CurrentInput::Get(mEndpoint, &currentInput);
+    if (Status::Success != status)
+    {
+        ChipLogError(Zcl, "Unable to save CurrentInput attribute, err:0x%x", to_underlying(status));
```

Comment:
What is "save" doing in this message?  We are not saving anything here...

Reply:
Fixed done. Thanks for the correction.